drivers/flash/spi_nor.c

```diff
@@ -1106,6 +1145,13 @@ static int spi_nor_configure(const struct device *dev)
 #endif /* CONFIG_FLASH_PAGE_LAYOUT */
 #endif /* CONFIG_SPI_NOR_SFDP_MINIMAL */
 
+#ifdef CONFIG_SPI_NOR_MX_HIGH_PERF_MODE
+	rc = mx_specific_enable_high_perf_mode(dev, jedec_id);
+	if (rc != 0) {
+		LOG_ERR("Enable high performace mode failed: %d", rc);
```

Comment:
I think the log line should go to the `mx_specific_enable_high_perf_mode` .
Till there is only one vendor and one device series having the feature, it can go to code of the vendor specific function.